src/libcore/result.rs

```diff
@@ -240,7 +240,14 @@ use self::Result::{Ok, Err};
 
 use clone::Clone;
 use fmt;
-use iter::{Iterator, DoubleEndedIterator, FromIterator, ExactSizeIterator, IntoIterator};
+use iter::{
+    Iterator,
+    FromIterator,
+    IntoIterator,
+    ExactSizeIterator,
+    DoubleEndedIterator,
+    FusedIterator,
+};
```

Comment:
I think our style is to currently split this across two lines instead of many, perhaps that can be done instead?